app/src/main/res/drawable/gradient.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is ok but I would like to change its name for something more representative. Remember that in an application we can have lots of gradients

Reply:
✔ **Done**

Name updated to _gradient_fragment_login_banner.xml_